bundle-workflow/src/paths/script_finder.py

```diff
@@ -64,8 +64,9 @@ def find_build_script(cls, component_name, git_dir):
     @classmethod
     def find_integ_test_script(cls, component_name, git_dir):
         paths = [
-            os.path.realpath(os.path.join(git_dir, "integtest.sh")),
-            os.path.realpath(os.path.join(git_dir, "scripts/integtest.sh")),
+            # TODO: Uncomment this after the integtest.sh tool is removed from plugin repos. See issue #497
+            # os.path.realpath(os.path.join(git_dir, "integtest.sh")),
+            # os.path.realpath(os.path.join(git_dir, "scripts/integtest.sh")),
```

Comment:
Are there any repos doing this? If not I would uncomment it, this way repos can actually customize the integtest.sh script. And remove the option of having it at the root permanently.

Reply:
yeah, the 5 mentioned in issue #497  are doing this. The idea was to unblock integration testing, while we get these resolved in parallel, inside the mentioned plugin repos.